README.md

```diff
@@ -1 +1,72 @@
-# bfx-hf-ui-config
\ No newline at end of file
+
+# bfx-hf-ui-config
+
+* This library gives a set of configs, which can be used in all `bfx-hf-*` services
+* Any config in `bfx-hf-ui-config` contain predefined default values
+  
+
+## Installation
+
+```bash
+
+npm i --save bfx-hf-ui-config
+
+```
+
+## Docs
+UserSettings (see `docs/user_settings.md`)  
+
+## Quickstart & Example
+
+  
+
+To get started, import an `bfx-hf-ui-config` to your project, then get any config  you need, for istance it will be UserSettings:
+
+  
+
+```js
+
+const  { _default }  =  require('bfx-hf-ui-config').UserSettings // get UserSettings default values
+
+  
+
+const send =  require('../../util/ws/send')
+
+const sendError =  require('../../util/ws/send_error')
+
+const isAuthorized =  require('../../util/ws/is_authorized')
+
+  
+
+module.exports  =  async  (server, ws, msg)  =>  {
+
+	const  {  db  }  =  server
+
+	const  {  UserSettings  }  =  db
```

Comment:
Please fix the spacing above and below, there are many redundant newlines.

Reply:
done